st3/sublime_lib/view_stream.py

```diff
@@ -117,12 +127,14 @@ def write(self, s):
         """
         old_size = self.view.size()
         self.view.run_command('insert', {'characters': s})
+        self._maybe_show_cursor()
         return self.view.size() - old_size
 
     def print(self, *objects, sep=' ', end='\n'):
         """Shorthand for :func:`print()` passing this ViewStream as the `file`
         argument."""
         print(*objects, file=self, sep=sep, end=end)
+        self._maybe_show_cursor()
```

Comment:
Should be redundant because `print` calls the `.write` method which already has this call.